cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -616,6 +619,276 @@ async def _get_trustworthiness_score_async(
             raise e
 
 
+class TLMHybrid:
```

Comment:
How about we move all the Hybrid specific stuff into new file like `trustworthy_language_model.hybrid`, WDYT?

I think it'd lead to less confusion/clutter since TLM and TLMHybrid have most of the same methods

Reply:
could be: `utils.tlm_hybrid`

Reply:
Moved it into `utils.tlm_hybrid`! Docs look like this now:
![image](https://github.com/user-attachments/assets/c3e3b31f-36ac-4ce9-9f6b-47086ef1b45e)

The only thing I can't fix is the **Tlm Hybrid** titile. It seems like the capitalization is automatic when the file is converted to md (I can't find a way for it to capitalize TLM in the configs), the only way I can fix it is to write a script to edit the MD file generated - is that something that we should do (feels quite hacky)?

Reply:
thanks this seems good. No worries about the capitalization, this is ok for now